docs/user/introduction.asciidoc

```diff
@@ -1,145 +1,447 @@
-is a better visualization for your data, *Lens* suggests it, allowing for quick
-switching between visualization types.
-
-Once your visualizations are just the way you want,
-use <<dashboard, Dashboard>> to collect them in one place. A dashboard provides
-insights into your data from multiple perspectives.
+Data exploration and visualizations are core features of {kib}.
+You can quickly search through large amounts of data, explore fields and values
+present, and then use {kib}’s drag-and-drop interface to rapidly build charts, tables, metrics, and more.
 
 [role="screenshot"]
 image::images/intro-dashboard.png[Sample eCommerce data set dashboard]
 
-{kib} also offers these visualization features:
+[float]
+==== Create visualizations
 
-* <<canvas, Canvas>> gives you the ability to present your data in a
-visually compelling, pixel-perfect report. Give your data the “wow” factor
-needed to impress your CEO or to captivate coworkers with a big-screen display.
+A typical data visualization journey looks like this, from adding your data to sharing your results.
 
-* <<maps, Maps>> enables you to ask (and answer) meaningful
-questions of your location-based data. Maps supports multiple
-layers and data sources, mapping of individual geo points and shapes,
-and dynamic client-side styling.
+[role="screenshot"]
+image::images/visualization-journey.png[User visualization journey]
+
+[[get-data-into-kibana]]
+. *Add data.* The best way to add {es} data to {kib} is to use one of our guided processes,
+available from the <<kibana-home-page,home page>>. You can collect data from an app or service or upload a
+file that contains your data. If you’re not ready to use your own data,
+you can add a sample data set and give {kib} a test drive.
+
+. *Explore.* With <<discover,*Discover*>>, you can search your data for hidden
+insights and relationships. Ask your questions, and then filter the results to just the data you want.
+Or, just see the most recent documents added to {es}.
 
-* <<tsvb, TSVB>> allows you to combine
-an infinite number of aggregations to display complex data.
-With TSVB, you can customize
-every aspect of your visualization. Choose your own date format and color
-gradients, and easily switch your data view between time series, metric,
-top N, gauge, and markdown.
+. *Visualize.* {kib} provides many options for visualizing your data&mdash;from simple
+aggregation-based charts to complex time series visualizations.
+<<dashboard, *Dashboard*>> is your starting point for building visualizations,
+and then pulling them together to show your data from multiple perspectives.
+
+. *Present.* <<canvas, *Canvas*>> enables you to display your data in a visually
+compelling, pixel-perfect presentation. With **Canvas**, you can give your data
+the “wow” factor needed to impress your CEO or to captivate coworkers with a big-screen display.
+
+. *Share.* Share your findings using the option that works best for you&mdash;embed
```

Comment:
Instead of listing out the specific sharing options, which we already do in the reporting docs, we need to cover the benefits of sharing to a larger audience.